docker/ds/schema/openidm.ldif

```diff
@@ -0,0 +1,115 @@
+#
```

Comment:
From a pure standard point of view, this file contains invalid LDIF.
All attributes of the same types should be next to each other. In other words, put all attributeTypes together, and then all objectClasses.

Reply:
Thanks Ludo!